### ISSUE TITLE
question about result on Kitty datasets

### ISSUE THREAD

**Issue by YWliu040120** [vita-epfl/FG2#10]
Hello ,Thanks for your amazing work ! Recently I want to reproduce your method on Kitty datasets ,I ran the command exactly as written in the README.md ：

python kitti_train_test.py -b 24 -t train

but I get a bad result ,which is far away from the result you updated in this repo。

the following is my training loss curve and final results

<img width="783" height="359" alt="Image" src="https://github.com/user-attachments/assets/2a1881d2-4a22-40d3-9501-81419c39d285" />

<img width="771" height="359" alt="Image" src="https://github.com/user-attachments/assets/42620256-35bf-4534-807c-4546a28c854d" />

<img width="750" height="362" alt="Image" src="https://github.com/user-attachments/assets/2206b768-ab42-4977-927d-e5d0ae4d5abd" />

<img width="974" height="1194" alt="Image" src="https://github.com/user-attachments/assets/b9c905d6-b787-4da5-8546-03147bc9a30f" />

and the following is the results you updated in README.md
<img width="1262" height="499" alt="Image" src="https://github.com/user-attachments/assets/d1744ccf-016e-4ea6-97e8-db28f51b28f5" />

**Comment by ZiminXia** [comment]
Hi, thank you for your interest in our work.

The training curves look quite different compared to the logs from my previous runs. In the meantime, I’ve started a new training run with the same settings to double-check.

As a quick sanity check, are you able to reproduce the KITTI test results? Also, which GPU are you using?

**Comment by YWliu040120** [comment]
> Hi, thank you for your interest in our work.
> 
> The training curves look quite different compared to the logs from my previous runs. In the meantime, I’ve started a new training run with the same settings to double-check.
> 
> As a quick sanity check, are you able to reproduce the KITTI test results? Also, which GPU are you using?

Thanks for your reply ! I git clone your code Around August 2025，so I forget to read the issues you closed about the bug in dataloader_kitti.py

<img width="1904" height="190" alt="Image" src="https://github.com/user-attachments/assets/fd215e04-8952-4ee4-9ec0-c14a7c3971b7" />
I have fixed the bug and carry out the same experiment,hope it will successfully reproduce the results !
I am sorry again for my fault ...
After the bug fixed , I can get the results in your repo with your released weights , hope my second training can get right result !
